.github/workflows/automated_release.yaml

```diff
@@ -0,0 +1,21 @@
+name: Automated release creation
+
+# This workflow triggers a centralized reusable workflow for managing the entire release process, which:
+# 1. Checks for the latest successful prerelease, excluding any labeled with "artifacts-pending" or "pre-release-failure",
+#    and promotes it to a full release if found suitable.
+# 2. Automatically triggers the creation of a new prerelease based on the specified criteria, fully automating the release cycle.
+#
+# The workflow is designed to be triggered from the default branch, supporting both automated and manual prerelease
+# creations for flexibility in release management.
```

Comment:
is this text needed?

Reply:
We could also _centralise_ this documentation by adding a link (only if we really think it is useful, considering the action being used below, it is pretty obvious): https://github.com/newrelic/coreint-automation/blob/main/.github/workflows/reusable_release_automation.yaml#L3-L9

Reply:
I added it for extra context and not having to jump into the reusable... but it's indeed explaining the reusable itself. I'm going to remove it.